nemo/collections/common/metrics/perf_metrics.py

```diff
@@ -68,6 +85,29 @@ def __init__(
 
         self.model = self.model.lower() if self.model is not None else self.model
 
+    def tb_logger(self, trainer: Trainer) -> TensorBoardLogger | None:
+        if not trainer.loggers:
+            return None
+        for logger in trainer.loggers:
+            if isinstance(logger, TensorBoardLogger):
+                return logger.experiment
+        return None
+
+    def wandb_logger(self, trainer: Trainer) -> WandbLogger | None:
+        if not trainer.loggers:
+            return None
+        for logger in trainer.loggers:
+            if isinstance(logger, WandbLogger):
+                return logger.experiment
+        return None
+
+    def setup(self, trainer, pl_module, stage):
+        """
+        PyTorch Lightning callback hook to calculate model FLOPs before fit/test
+        """
+        self._tb_logger = self.tb_logger(trainer)
+        self._wandb_logger = self.wandb_logger(trainer)
```

Comment:
Can I user enable both TB and Wandb loggers ?

Reply:
If a user can only enable one logger, we could just have a single logger that either is a TB or Wandb or Neptune or etc. (Union[]). 

Technically if  somebody could have both, this is good.

Reply:
I think we would still need to check which logger we are using as the logging API still differs- `.add_scalar()` for TB `.log({key: value})` for WandB, etc.